insights-notification-schemas-java/src/main/java/com/redhat/cloud/notifications/ingress/ParsingException.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.cloud.notifications.ingress;
+
+import com.networknt.schema.ValidationMessage;
+
+import java.util.Set;
+
+public class ParsingException extends RuntimeException {
+
+    private Set<ValidationMessage> validationMessages;
```

Comment:
Would the validation messages follow the structure of the incoming JSON if this was a `List` instead of a `Set`?

Reply:
I don't think so - this `set` comes from the validation library and is copied to the exception as is